deployment/CP05MOAS-A6264_Deploy.csv

```diff
@@ -1,99 +1,99 @@
 WJ20190617,,WJ20190617,,CP05MOAS-A6264-05-ADCPAN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.93057,-70.90167,,,,abbreviated mission due to time constraints (AC-1 to AC-2)
 WJ20190617,,WJ20190617,,CP05MOAS-A6264-06-PARADN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50180,39.93057,-70.90167,,,,abbreviated mission due to time constraints (AC-1 to AC-2)
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-00-ENG000000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,40.34183,-70.89003,,,,across-shelf survey
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-01-FLORTN000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01109,40.34183,-70.89003,,,,across-shelf survey
-WJ20190709,,WJ20190709,,CP05MOAS-A6264-02-DOSTAN000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01520,40.34183,-70.89003,,,,across-shelf survey
+WJ20190709,,WJ20190709,,CP05MOAS-A6264-02-DOSTAN000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01519,40.34183,-70.89003,,,,across-shelf survey
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-03-CTDAVN000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00425,40.34183,-70.89003,,,,across-shelf survey
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-04-NUTNRN000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,40.34183,-70.89003,,,,across-shelf survey
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-05-ADCPAN000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,40.34183,-70.89003,,,,across-shelf survey
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-06-PARADN000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50180,40.34183,-70.89003,,,,across-shelf survey
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-00-ENG000000,13,1,2019-07-11T13:50:00,2019-07-11T23:30:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,40.33785,-70.904967,,,,custom bottom boundary layer mission on across-shelf line
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-01-FLORTN000,13,1,2019-07-11T13:50:00,2019-07-11T23:30:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01109,40.33785,-70.904967,,,,custom bottom boundary layer mission on across-shelf line
-WJ20190709,,WJ20190709,,CP05MOAS-A6264-02-DOSTAN000,13,1,2019-07-11T13:50:00,2019-07-11T23:30:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01520,40.33785,-70.904967,,,,custom bottom boundary layer mission on across-shelf line
+WJ20190709,,WJ20190709,,CP05MOAS-A6264-02-DOSTAN000,13,1,2019-07-11T13:50:00,2019-07-11T23:30:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01519,40.33785,-70.904967,,,,custom bottom boundary layer mission on across-shelf line

```

Comment:
Looks good. Associated checkout document .zip file will need to be revised - remove the 1520 cal doc and insert the 1519 cal doc. (3403-40001-00020-A.zip) [zip file also still references previous cruise P12b]